### PR TITLE
add report-content-imagedata-not-num

### LINKED ISSUE #3540: Report on non-SVG with non-integer height or width
Create a report for any objects which contain content metadata that has `<file[@mimetype!="image/svg+xml">` and child `<imageData>` has a height or width attribute that is not blank and not an integer.

See #3323

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

To look beyond the tea leaves into the actual data, I presume.

Closes #3540 

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

I ran the code in this report against some contentMetadata in the rails console.

### CODE REVIEW

File: bin/reports/report-content-imagedata-not-num [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+require_relative '../../lib/report'
+
+Report.new(name: 'content-missing-imagedata-non-num', dsid: 'contentMetadata').run do |ng_xml|
+  not_svg = '@mimetype!="image/svg+xml"'
+
+  width = ng_xml.xpath("/contentMetadata/resource/file[#{not_svg}]/imageData/@width").presence&.first&.content
+  height = ng_xml.xpath("/contentMetadata/resource/file[#{not_svg}]/imageData/@height").presence&.first&.content
+
+  # we are interested in the ones that cannot be parsed as a Float value
+  Float(width) if width.present?
```
Comment (justinlittman) [inline bin/reports/report-content-imagedata-not-num:14]: Integer, not float.

**Review by justinlittman — CHANGES_REQUESTED** [review]

File: bin/reports/report-content-imagedata-not-num (hunk 1 above)
Comment (ndushay) [reply bin/reports/report-content-imagedata-not-num:14]: er ... we have ones that are Float like 29.82334124mm -- I thought that was ok

Comment (ndushay) [reply bin/reports/report-content-imagedata-not-num:14]: Is the goal to identify all the non-integer ones, or the ones with  "mm" or whatever?

Comment (justinlittman) [reply bin/reports/report-content-imagedata-not-num:14]: Non-integer and non-blank.

Comment (ndushay) [reply bin/reports/report-content-imagedata-not-num:14]: ok!  can do.

**Review by justinlittman — APPROVED** [review]

**Pull request merged**